src/pages/workflows/workflow/WorkflowDetails.js

```diff
@@ -215,9 +215,7 @@ const WorkflowConfigs = () => {
           rowCount: allConfigs.length,
           columns: [
             {
-              headerRenderer: () => span({
-                'aria-label': 'warnings'
-              }),
+              headerRenderer: () => div({ className: 'sr-only' }, ['Warnings']),
```

Comment:
If it's a `div` now instead of a `span` should there be an `aria-label`? Or does the `sr-only` class name accomplish the same thing?

Reply:
`sr-only` accomplishes the same thing. `aria-label` from my understanding is specifically for like… links/inputs/etc (interactive elements).

What `sr-only` does is basically hides the element, but because it is present still, screen readers will pick it up!